src/cli/test_command.zig

```diff
@@ -175,9 +175,18 @@ pub const CommandLineReporter = struct {
 
         var this: *CommandLineReporter = @fieldParentPtr("callback", cb);
 
-        writeTestStatusLine(.pass, &writer);
-
-        printTestLine(label, elapsed_ns, parent, false, writer);
+        const debug_jest = blk: {
```

Comment:
why not bun.Output.Scoped(.jest, false).isVisible()

(captial `S` to get the namespace instead of the log function)

Reply:
didnt know about that function, good callout